src/main/java/edu/tamu/app/service/manager/GitHubMilestoneService.java

```diff
@@ -0,0 +1,343 @@
+package edu.tamu.app.service.manager;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+
+import org.apache.log4j.Logger;
+import org.kohsuke.github.GHIssue;
+import org.kohsuke.github.GHIssueState;
+import org.kohsuke.github.GHLabel;
+import org.kohsuke.github.GHMilestone;
+import org.kohsuke.github.GHMilestoneState;
+import org.kohsuke.github.GHOrganization;
+import org.kohsuke.github.GHProject;
+import org.kohsuke.github.GHProject.ProjectStateFilter;
+import org.kohsuke.github.GHProjectCard;
+import org.kohsuke.github.GHProjectColumn;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GHUser;
+import org.kohsuke.github.GitHub;
+import org.kohsuke.github.GitHubBuilder;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import edu.tamu.app.cache.model.Card;
+import edu.tamu.app.cache.model.Member;
+import edu.tamu.app.cache.model.RemoteProject;
+import edu.tamu.app.cache.model.Sprint;
+import edu.tamu.app.model.ManagementService;
+import edu.tamu.app.model.request.FeatureRequest;
+import edu.tamu.app.rest.TokenAuthRestTemplate;
+
+public class GitHubMilestoneService extends MappingRemoteProjectManagerBean {
+
+    private static final Logger logger = Logger.getLogger(GitHubProjectService.class);
+
+    protected static final String ORGANIZATION = "TAMULib";
```

Comment:
I suggest creating an abstract github class (or similar) and moving the common code, such as these defines and other identical functions between the Milestone and Project service implementations.